plugins/source/digitalocean/codegen/resources.go

```diff
@@ -0,0 +1,373 @@
+package main
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
+	"github.com/cloudquery/cloudquery/plugins/source/digitalocean/codegen/recipes"
+	"github.com/cloudquery/cloudquery/plugins/source/digitalocean/resources/services/spaces"
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/digitalocean/godo"
+)
+
+var Resources = []*recipes.Resource{
+	{
+		Service: "account",
```

Comment:
For consistency with other tables, should this maybe be called `accounts` (plural)?